requirements/edx-sandbox/pre.txt

```diff
@@ -0,0 +1,7 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    make upgrade
+#
+numpy==1.6.2
```

Comment:
nit: please remove [`numpy==1.6.2` from edx-sandbox/base.txt](https://github.com/open-craft/edx-platform/blob/b073ee9de0d307046e9243bfeed414f2bff655de/requirements/edx-sandbox/base.txt#L25) (and `base.in`).

Reply:
@jill, `edx-sandbox/base.txt` is used in https://github.com/open-craft/edx-platform/blob/opencraft-release/ironwood.2/requirements/monitoring/sandbox-requirements.txt. So I think we should leave it as-is.

Reply:
Ok sure.